backend/api/tests/test_auth.py

```
@@ -0,0 +1,56 @@
+    """Return the result list from paginated responses."""
+
+    if isinstance(payload, dict) and "results" in payload:
+        return payload["results"]
+    return payload
+
+
+@pytest.mark.django_db
+def test_jwt_authentication_flow(api_client, user_factory):
+    """Exercise the token obtain/refresh endpoints exposed by the API."""
+
+    password = "Str0ngPass!"
```

Ruff (0.14.0)

Possible hardcoded password assigned to: "password"

(S105)